document/core/binary/instructions.rst

```diff
@@ -226,10 +226,10 @@ Each variant of :ref:`memory instruction <syntax-instr-memory>` is encoded with
      \hex{3E}~~m{:}\Bmemarg &\Rightarrow& \I64.\STORE\K{32}~m \\ &&|&
      \hex{3F}~~x{:}\Bmemidx &\Rightarrow& \MEMORYSIZE~x \\ &&|&
      \hex{40}~~x{:}\Bmemidx &\Rightarrow& \MEMORYGROW~x \\ &&|&
-     \hex{FC}~~8{:}\Bu32~~y{:}\Bdataidx~x{:}\Bmemidx &\Rightarrow& \MEMORYINIT~x~y \\ &&|&
-     \hex{FC}~~9{:}\Bu32~~x{:}\Bdataidx &\Rightarrow& \DATADROP~x \\ &&|&
-     \hex{FC}~~10{:}\Bu32~~x{:}\Bmemidx~~y{:}\Bmemidx &\Rightarrow& \MEMORYCOPY~x~y \\ &&|&
-     \hex{FC}~~11{:}\Bu32~~x{:}\Bmemidx &\Rightarrow& \MEMORYFILL~x \\
+     \hex{FC}~~8{:}\Bu32~~src{:}\Bdataidx~dst{:}\Bmemidx &\Rightarrow& \MEMORYINIT~dst~src \\ &&|&
+     \hex{FC}~~9{:}\Bu32~~dst{:}\Bdataidx &\Rightarrow& \DATADROP~dst \\ &&|&
```

Comment:
Multi-letter variables would need the `\X{...}` macro for correct type setting. However, the use of meta-variables is heavily based on defined naming conventions, and only x, y are introduced for indices. I'd rather not introduce one-off non-standard meta-variables, we don't do that in other similar places either (cf. table.copy).

Reply:
Ack, I reverted this change.